WooCommerce/Classes/Authentication/Epilogue/SwitchStoreUseCase.swift

```diff
@@ -146,6 +146,10 @@ final class SwitchStoreUseCase: SwitchStoreUseCaseProtocol {
     }
 
     private func refreshStoredSites() {
-        try? resultsController.performFetch()
+        do {
+            try resultsController.performFetch()
+        } catch {
+            DDLogError("Unable to refresh stored sites: \(error) ")
```

Comment:
Similar to the above comment:
```suggestion
            DDLogError("⛔️ Unable to refresh stored sites: \(error) ")
```